### PR TITLE
COM-331 Update schema add business address resolver

### PR DESCRIPTION
**Pull request by sinahashemy**
Update js-sdk to add a new Graph-QL resolver for the business address.

**Addresses [COM-331](https://kontist.atlassian.net/browse/COM-331) , [COM-332](https://kontist.atlassian.net/browse/COM-332)**

### CODE REVIEW

File: tests/graphql/businessAddress.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        createBusinessAddress: businessAddressData,
+      } as any);
+
+      // act
+      const result = await businessAddress.create({
+        payload: {
+					street: "Riccarton Av. 1212",
+					postCode: "RD88990",
+					city: "Christpost",
+					movingDate: new Date("2019-01-01"),
```
Comment (dmytroyarmak) [inline tests/graphql/businessAddress.spec.ts:104]: 🔧  Indentations is strange here

File: lib/graphql/businessAddress.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   * Creates a business address
+   *
+   * @param args query parameters
+   * @returns the newly created business address
+   */
+  public async create(args: MutationCreateBusinessAddressArgs): Promise<BusinessAddressModel> {
+    const result = await this.client.rawQuery(CREATE_BUSINESS_ADDRESS_MUTATION, args);
+    return result.createBusinessAddress;
+  }
+}
```
Comment (dmytroyarmak) [inline lib/graphql/businessAddress.ts:91]: 🔧  Please make sure that there is newline at the end of the file

**Review by dmytroyarmak — APPROVED** [review]
👍

File: tests/graphql/businessAddress.spec.ts (hunk 1 above)
Comment (sinahashemy) [reply tests/graphql/businessAddress.spec.ts:104]: I found the reason and corrected it.

**Pull request merged**